test/OpenSpaceToolkit/Astrodynamics/NumericalSolver.test.cpp

```diff
@@ -98,6 +130,22 @@ class OpenSpaceToolkit_Astrodynamics_NumericalSolver : public ::testing::Test
     }
 };
 
+class OpenSpaceToolkit_Astrodynamics_NumericalSolver_Parametrized
+    : public OpenSpaceToolkit_Astrodynamics_NumericalSolver,
+      public ::testing::WithParamInterface<Tuple<NumericalSolver::StepperType>>
+{
+};
+
+INSTANTIATE_TEST_SUITE_P(
```

Comment:
This is super cool. Didn't know it existed. 

Just to make sure I'm reading stuff right, it looks like we're now testing all of the numerical solvers against the same tolerance. Is that correct?

Reply:
Yes that's right. By reducing the step size on the RK54 we can compare everything with the same tolerance. Which is a small change from before. Great 👁️

Reply:
Looks like the tolerance is also looser in some places. I think this is fine for this class, since they are just wrappers over the boost numerical solver classes. 

If we ever decide that we do need to test higher tolerances in some places, is that something we can include in the Google Test parameterization?

Reply:
Yeah for sure, we can just add it as an additional parameter in the parametrized template. Tagging you where.

Reply:
Sick. This is cool w me then